packages/open-api-page/server.js

```diff
@@ -27,10 +27,15 @@ function getPropsForPage(schema, params) {
       path: getServicePathSlug(serviceId),
     }
   })
+  // If there is only a single service,
+  // we'll display that service on the landing page
   const isSingleService = navData.length == 1
+  // For the single service use case,
+  // we also ensure the single sidenav link points to the landing page
+  // rather than a specific operation-slug-based path as it would otherwise
+  if (isSingleService) navData[0].path = ''
```

Comment:
This fixes an issue where on https://cloud.hashicorp.com/api-docs/packer, if you click the single already-highlighted sidebar item, you end up on `/api-docs/packer/packer-service`, which is a 404. With this change, the link will more correctly just link to same page (`/api-docs/packer`).